actors/runtime/src/test_utils.rs

```diff
@@ -785,7 +785,7 @@ impl Runtime<MemoryBlockstore> for MockRuntime {
         F: FnOnce(&mut C, &mut Self) -> Result<RT, ActorError>,
     {
         if self.in_transaction {
-            return Err(actor_error!(SysErrIllegalActor; "nested transaction"));
+            return Err(actor_error!(unspecified; "nested transaction"));
```

Comment:
Assertion failed here and at 813 and all the "self.in_transaction" checks

Reply:
fixed